connectors/src/connectors/github/temporal/activities.ts

```diff
@@ -281,7 +281,7 @@ export async function githubUpsertDiscussionActivity(
     documentUrl: discussion.url,
     timestampMs: new Date(discussion.createdAt).getTime(),
     tags,
-    parents: [],
+    parents: [discussionNumber.toString(), repoId.toString()],
```

Comment:
documentId should be visible here

Reply:
Otherwise we risk divergence?

Reply:
Actually maybe it's not documentid but at least should be a variable that is used to set the ressource internal Id ?

Reply:
Since no valid external id exist I will use the pre-existing getIssueDocumentId here and document it; same for discussion